### DISCUSSION TITLE
When will NextJS 13 ready for prod?

### DISCUSSION THREAD

**Discussion by Adityak4201** [vercel/next.js#47956] [Help]
### Summary

Can anyone help me when will NextJS 13 be ready and stable for production servers?

**Comment by philwolstenholme** [comment]
See https://github.com/vercel/next.js/discussions/47925

No one can say for sure, probably not even Vercel as there are probably some `app` directory features that are waiting on changes or information from the React core team. It's up to you to figure out the pros and cons for your specific project and team.
